staging/src/github.com/kubeedge/mapper-framework/pkg/util/parse/grpc.go

```diff
@@ -172,6 +176,50 @@ func buildPropertiesFromGrpc(device *dmiapi.Device) []common.DeviceProperty {
 					klog.Errorf("err: %+v", err)
 					return nil
 				}
+				dbconfig = common.DBConfig{
+					Influxdb2ClientConfig: clientconfig,
+					Influxdb2DataConfig:   dataconfig,
+				}
+			case "redis":
+				clientConfig, err := json.Marshal(pptv.PushMethod.DBMethod.Redis.RedisClientConfig)
+				if err != nil {
+					klog.Errorf("err: %+v", err)
+					return nil
+				}
+				dbconfig = common.DBConfig{
+					RedisClientConfig: clientConfig,
+				}
+			case "tdengine":
+				clientConfig, err := json.Marshal(pptv.PushMethod.DBMethod.Tdengine.TdEngineClientConfig)
+				if err != nil {
+					klog.Errorf("err: %+v", err)
+					return nil
+				}
+				dbconfig = common.DBConfig{
+					TDEngineClientConfig: clientConfig,
+				}
+			}
```

Comment:
It seems that you made an error during rebase. The case here should be the switch of dbMethodName instead of the switch of pushMethodName.
![微信截图_20231128154121](https://github.com/kubeedge/kubeedge/assets/48212997/197d52d5-f0d9-49dd-8696-65d89dfe040b)